sensor_msgs/Image[] pos_image_patches
sensor_msgs/Image[] neg_image_patches

# Descriptor[] pos_descs
# Descriptor[] neg_descs